kernel/dev/rtc/rtc.cpp

```diff
@@ -47,17 +56,20 @@ void rtc_init() {
     register_interrupt_handler(IRQ8, rtc_callback);
 }
 
-void rtc_callback(registers_t *regs) {
+void rtc_callback(registers_t* regs)
```

Comment:
Forgot to mention this earlier -- any function not defined in the header should be static. Besides this and the tabs, we're good!